analysers/experiments/executionTime.py

```diff
@@ -0,0 +1,73 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from datetime import timedelta
+
+import scipy.integrate as integrate
+import scipy.special as special
+import numpy as np
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+def createQuery(dataRDD, experimentID):
+    from commons import computeMode, computeMetrics
+    
+    queries = []
+    
+    processes = dataRDD.map(lambda a: a["process_definition_id"]).distinct().collect()
+    
+    for process in processes:
+        filteredRDD = dataRDD.filter(lambda a: a["process_definition_id"] == process).cache()
+        
+        mode = computeMode(filteredRDD.map(lambda r: (r['execution_time'], 1)))
+    
+        data = filteredRDD.map(lambda r: r['execution_time']).collect()
```

Comment:
@VincenzoFerme computing mode is different, so I prep the RDD differently (mode we use Spark for everything, not numpy)